com.fidele007.akkhara.yaml

```diff
@@ -0,0 +1,79 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.fidele007.akkhara
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: akkhara
+separate-locales: false
+finish-args:
+  - --share=network # access the network
+  - --share=ipc # share IPC namespace with the host (necessary for X11)
+  - --socket=wayland # show windows using wayland
+  - --socket=fallback-x11 # fallback to x11 if wayland is not available
+  - --socket=pulseaudio # Play sound with PulseAudio (?)
+  - --device=dri # GPU acceleration if needed
+
+modules:
+  - name: libass
+    # mpv won't build without this
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://github.com/libass/libass/releases/download/0.15.0/libass-0.15.0.tar.gz
+        sha256: 9cbddee5e8c87e43a5fe627a19cd2aa4c36552156eb4edcf6c5a30bd4934fe58
+  - name: mpv
+    buildsystem: simple
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+    build-commands:
+      - python3 waf configure --prefix=/app --enable-libmpv-shared --disable-build-date
+        --disable-manpage-build --disable-alsa --enable-libarchive
+        --disable-lua --disable-javascript --disable-uchardet --disable-drm --disable-dvdnav
+      - python3 waf build
+      - python3 waf install
+    post-install:
+      # save screenshots at ~/Pictures/mpv
+      - echo "screenshot-directory=~/Pictures/mpv" > /app/etc/mpv/mpv.conf
+    sources:
+      - type: archive
+        url: https://github.com/mpv-player/mpv/archive/v0.35.0.tar.gz
+        sha256: dc411c899a64548250c142bf1fa1aa7528f1b4398a24c86b816093999049ec00
+      - type: file
+        url: https://waf.io/waf-2.0.22
+        sha256: 0a09ad26a2cfc69fa26ab871cb558165b60374b5a653ff556a0c6aca63a00df1
+        dest-filename: waf
+  - name: akkhara
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      # Extract release build
+      - mkdir -p akkhara
+      - tar -xf akkhara_*.tar.gz -C akkhara
+      # Copy all built files to Flatpak-based location
+      - cp -r akkhara /app/
+      - chmod +x /app/akkhara/akkhara
+      - mkdir -p /app/bin
+      - ln -s /app/akkhara/akkhara /app/bin/akkhara
+      # Install the icon
+      - mkdir -p /app/share/icons/hicolor/scalable/apps
+      - cp com.fidele007.akkhara.svg /app/share/icons/hicolor/scalable/apps/
+      # Install the desktop file
+      - mkdir -p /app/share/applications
+      - cp com.fidele007.akkhara.desktop /app/share/applications/
+      # Install the AppStream metadata file.
+      - mkdir -p /app/share/metainfo
+      - cp com.fidele007.akkhara.metainfo.xml /app/share/metainfo/
+    sources:
+      - type: file
+        url: https://github.com/fidele007/akkhara-support/releases/download/1.2.0/akkhara_1.2.0_amd64.tar.gz
```

Comment:
so what happen in the `aarch64` package?

Reply:
why not building from source?

Reply:
> why not building from source?

Why?

Reply:
Because the code is hosted on a private repository. Is it a requirement to build from source for the PR to be accepted?

Reply:
You need this https://docs.flathub.org/docs/for-app-authors/maintenance#only-build-on-x86_64

Reply:
The `flathub.json` file is added.